src/Faithlife.Tracing.AspNetWebApi/Faithlife.Tracing.AspNetWebApi.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <TargetFrameworks>net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Faithlife.Tracing.AspNetMvc\AspNetTracing.cs" />
+    <Compile Include="..\Faithlife.Tracing.AspNetMvc\RequestActionTraceProvider.cs" />
```

Comment:
Hm. What happens if a project references both `AspNetWebApi` and `AspNetMvc`? I know you're trying to avoid another shared `AspNet` dependency, but is it really worth it?

Reply:
Those classes are both `internal` (implementation reuse without adding another dependency). All `public` types are unique.

Reply:
I guess I didn't realize it was safe to have two assemblies with the same internal types in the same namespace. But I've never tried it, either.

Would it be reasonable to put the `.cs` files in a sibling `Faithlife.Tracing.AspNet` directory without a `.csproj` and link to them in the same way from both assemblies?